src/main/js/main_html5.js

```diff
@@ -530,7 +533,7 @@ require('../../../html5-common/js/utils/environment.js');
           // TODO: Handle MUTED/UNMUTED_PLAYBACK_SUCCEEDED/FAILED in environments that do not support play promises.
           // Right now this is not needed because environments that do not support play promises do not have
           // autoplay restrictions.
-          if (typeof playPromise.catch === 'function') {
+          //if (typeof playPromise.catch === 'function') {
```

Comment:
please delete all the commented lines =)

Reply:
As far as I remember this check was added to avoid runtime error in case if `catch` or `then` doesn't exist.

Reply:
Oops, this is reverted now :)